classes/ActionScheduler_AdminView.php

```diff
@@ -35,9 +35,53 @@ public function init() {
 			} else {
 				add_action( 'admin_menu', array( $self, 'register_menu' ) );
 			}
+			add_action( 'admin_notices', array( $self, 'check_past_due_actions' ) );
 		}
 	}
 
+	/**
+	 * Check if there are any past due actions.
+	 *
+	 * This method will check if there are past due actions. Because Action Scheduler is action agnostic,
+	 * it will trigger an action with a list of hooks that callbacks can use to handle past due actions
+	 * as needed.
+	 *
+	 * To narrow down possible results, and improve query looking for past due actions, any plugin
+	 * must add their hook name to an array through the `action_scheduler_past_due_hooks`
+	 * filter.
+	 *
+	 * The action owner should be listening to the `action_schedule_past_due_action` action, 
+	 * and do something to warn the users about how to fix their WP-Cron or to run their actions manually.
+	 *
+	 * This method is executed on the `admin_notices`, so it is safe to render the admin notification.
+	 */
+	public function check_past_due_actions() {
+		$instance = ActionScheduler_Store::instance();
```

Comment:
`$instance` is a generic word - any object can be an "instance", so it doesn't give any indication what this variable contains.

In other locations, the store instance is assigned to a variable named `$store`, which more accurately described its contents. `ActionScheduler::store() is also used to access the instance, for simplicities sake (although it is really just an alias).

IMO, as the store is only being used twice, it doesnt' really need a variable at all, you could call `ActionScheduler::store()->query_actions()` and `ActionScheduler::store()-> fetch_action()` directly and save having to use a variable, while also making it clear what is being used.

Reply:
+1 to avoiding storing `ActionScheduler_Store::instance()` in a variable. Realistically, I'd like to see the instance we need be injected into the class constructor, but that's way out of scope for this PR 😃

Reply:
> IMO, as the store is only being used twice, it doesnt' really need a variable at all, you could call ActionScheduler::store()->query_actions() and ActionScheduler::store()-> fetch_action() directly and save having to use a variable, while also making it clear what is being used.
 
Works for me.